realm/src/androidTest/java/io/realm/tests/api/RealmListTest.java

```diff
@@ -1,195 +0,0 @@
-/*
```

Comment:
We could probably migrate some of the RealmListTests, All the RealmTests seem outdated however.